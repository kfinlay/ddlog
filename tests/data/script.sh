this_is_a_processing_script
