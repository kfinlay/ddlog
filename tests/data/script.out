This is the logfile from the processing script.
